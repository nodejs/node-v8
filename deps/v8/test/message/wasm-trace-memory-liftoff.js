// Copyright 2017 the V8 project authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

// Flags: --no-stress-opt --trace-wasm-memory --liftoff --no-future
// Flags: --no-wasm-tier-up --experimental-wasm-simd
// Flags: --enable-sse3 --enable-sse4-1

// Force enable sse3 and sse4-1, since that will determine which execution tier
// we use, and thus the expected output message will differ.
load("test/message/wasm-trace-memory.js");
