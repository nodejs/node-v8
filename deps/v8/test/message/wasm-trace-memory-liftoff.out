liftoff     func:     0+0x3     load from 0000000000000004 val: i32:0 / 00000000
liftoff     func:     1+0x3     load from 0000000000000001 val:  i8:0 / 00
liftoff     func:     3+0x5      store to 0000000000000004 val: i32:305419896 / 12345678
liftoff     func:     0+0x3     load from 0000000000000002 val: i32:1450704896 / 56780000
liftoff     func:     1+0x3     load from 0000000000000006 val:  i8:52 / 34
liftoff     func:     2+0x3     load from 0000000000000002 val: f32:68169720922112.000000 / 56780000
liftoff     func:     4+0x5      store to 0000000000000004 val:  i8:171 / ab
liftoff     func:     0+0x3     load from 0000000000000002 val: i32:1454047232 / 56ab0000
liftoff     func:     2+0x3     load from 0000000000000002 val: f32:94008244174848.000000 / 56ab0000
liftoff     func:     6+0x7      store to 0000000000000004 val: s128:48879 48879 48879 48879 / 0000beef 0000beef 0000beef 0000beef
liftoff     func:     5+0x3     load from 0000000000000002 val: s128:-1091633152 -1091633152 -1091633152 -1091633152 / beef0000 beef0000 beef0000 beef0000
liftoff     func:     7+0x3     load from 0000000000000004 val: i16:48879 / beef
liftoff     func:     8+0x3     load from 0000000000000002 val: i64:-4688528683866062848 / beef0000beef0000
liftoff     func:     9+0x3     load from 0000000000000002 val: f64:-0.000015 / beef0000beef0000
